app/controllers/api/v1/portfolio_items_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
           collection(Tag.find(params.require(:tag_id)).portfolio_items)
         else
           authorize(PortfolioItem)
-          collection(policy_scope(PortfolioItem.all))
+          collection(policy_scope(PortfolioItem.all), true)
```

Comment:
Can we use keyword arguments? e.g. `pre_authorized: true` instead of a bare boolean?